apps/mudita-center-e2e/src/page-objects/home.page.ts

```diff
@@ -7,12 +7,71 @@ import Page from "./page"
 
 class HomePage extends Page {
   get homeHeader() {
-    return $("h2*=Welcome to Mudita Center")
+    return $("h1*=Welcome to Mudita Center")
   }
 
   get notNowButton() {
     return $("button*=Not now")
   }
-}
 
+  get myDevicesDoesntShowButton() {
```

Comment:
The getter name does not correspond to the return value. The name suggests a button, but you are returning a paragraph. 

The same comment applies to other getters. (some of them are missing the type of returned element)

Reply:
Now it is really the button. 

Others fixed